src/layouts/GeneralLayout.vue

```diff
@@ -46,4 +46,32 @@ export default {
   top:0;
   left:0;
 }
-</style>
\ No newline at end of file
+
+@media screen and (min-width: 600px) {
```

Comment:
Let's consistently use the tailwind breakpoints. You can do
```suggestion
@screen md
```